src/views/Project/Details/General/index.js

```diff
@@ -43,7 +41,7 @@ import {
 
 import _ts from '#ts';
 
-import StakeholdersModal from './StakeholdersModal';
+import AddStakeholdersButton from '#components/general/AddStakeholdersButton';
```

Comment:
Lets duplicate this and create a new deep ui look alike design for stakeholders modal